app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -229,6 +230,13 @@ open class FenixApplication : LocaleAwareApplication() {
         // no-op, LeakCanary is disabled by default
     }
 
+    // This is for issue https://github.com/mozilla-mobile/fenix/issues/11660. We prefetch our info for startup
```

Comment:
I think it's confusing that we say, "our info" but really mean "top sites" so I'd rather say "We prefetch our top sites for startup" and it can be changed if the code becomes more general later.